src/v/cluster/self_test_rpc_types.h

```diff
@@ -308,6 +324,7 @@ struct start_test_request
     std::vector<diskcheck_opts> dtos;
     std::vector<netcheck_opts> ntos;
     std::vector<cloudcheck_opts> ctos;
+    std::vector<unknown_check> unknown_checks;
```

Comment:
Ah we should've done this with the cloud check opt too, but we need to bump the version in the serde envelope

Reply:
good spot

Reply:
Fixed.